flexmeasures/cli/data_add.py

```diff
@@ -286,7 +291,12 @@ def add_asset_type(**args):
     type=LongitudeField(),
     help="Longitude of the asset's location",
 )
-@click.option("--account-id", type=int, required=True)
+@click.option(
+    "--account-id",
+    type=int,
+    required=False,
+    help="Add asset to this account. Follow up with the account's ID. If not set, the asset will become public (which makes it accessible to all users).",
+)
```

Comment:
Should we print a INFO log in this command when a public asset has been created?
To make one, all one has to do is to leave off the account ID, after all. But it can have quite some data security effects.

Reply:
Probably a warning even.

Reply:
I added it.